subprojects/core/src/integTest/groovy/org/gradle/NativeServicesIntegrationTest.groovy

```diff
@@ -57,7 +57,7 @@ class NativeServicesIntegrationTest extends AbstractIntegrationSpec {
 
     def "native services are #description with systemProperties == #systemProperties"() {
         given:
-        executer.requireOwnGradleUserHomeDir().withNoExplicitNativeServicesDir()
+        executer.withGradleUserHomeDir(tmpDir.testDirectory).withNoExplicitNativeServicesDir()
```

Comment:
🤔 My understanding is that this is still in the test project home. Is it not?

Reply:
And project dir uses instance defined in:
https://github.com/gradle/gradle/blob/6e60ccd6e76735f829482c3ecd49af5f328789ad/testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/AbstractIntegrationSpec.groovy#L76

Reply:
Ah, I see. Can you leave a comment in the test what is going on?

As long as this works I am fine with it.

Reply:
Good idea, added a comment 👍